elements/rh-navigation-secondary/rh-navigation-secondary.ts

```diff
@@ -234,6 +243,20 @@ export class RhNavigationSecondary extends LitElement {
     }
   }
 
+  #onSlotchange() {
+    if (this._nav) {
+      this._nav.forEach(nav => {
+        this.#navItems = Array.from(
+          nav.querySelectorAll(':is(rh-navigation-secondary-dropdown, rh-secondary-nav-dropdown) > a, [slot="nav"] > li > a')
+        );
+      });
+      if (!this.#navItems) {
+        return;
+      }
+      this.#tabindex.initItems(this.#navItems);
```

Comment:
```suggestion
    this.#tabindex.initItems(this.#navItems ?? []);
```

six of one... 
this comment is purely aesthetic, feel free to pass over it